### ISSUE TITLE
Error object thrown by an incomplete request doesn't have relevant information

### ISSUE THREAD

**Issue by sreenaths** [jira-node/node-jira-client#356]
In the latest version 8.2.0, an incomplete request results in error without relevant information.

For instance I tried calling addNewIssue without passing an issuetype. Earlier this type of request was failing with an error that said "issue type is required".
But now, following is the error object that I received. 
```
[start:server] Error: [object Object]
[start:server]     at JiraApi.doRequest (/Users/ssomarajapuram/src/github/smart-ci/node_modules/jira-client/lib/jira.js:347:13
[start:server]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Reverting to 8.0.0 resolved the issue. It returns the required info, but in exchange it doesn't throw an error now.

**Comment by Seth10001** [comment]
Hi @sreenaths!

This was fixed in #353, but I haven't deployed it yet. I'll make a new release for it.

**Comment by Seth10001** [comment]
This should be fixed in 8.2.1, so I will close for now. Let me know if you are still seeing the issue
